guides/elements.md

```diff
@@ -480,3 +531,25 @@ de:
       contact_form:
         color: Button Farbe
 ~~~
+
+## Notes on interacting with Elements from the Admin interface
+
+While this guide is intended for developers building a CMS site, many developer will need to train clients and customers, or interact with the admin interface themselves.
```

Comment:
I don't think this paragraph should be part of the guides. This sounds like an explanation why this section is placed here. Can we remove it?

Reply:
Removed